test/whisperkit_test.py

```diff
@@ -0,0 +1,191 @@
+#
+# For licensing see accompanying LICENSE.md file.
+# Copyright (C) 2023 Argmax, Inc. All Rights Reserved.
+#
+
+import os
+import json
+import unittest
+import argparse
+import subprocess
+import concurrent.futures as futures
+
+from pathlib import Path
+from huggingface_hub import hf_hub_download
+from whisper.tokenizer import get_tokenizer
+from android_test_utils import AndroidTestsMixin, TestRunADB
+
+OUTPUT_FOLDER = './json/'
+TEST_DATASET = "librispeech-10mins" 
+TEST_FILES = [
+    "121-121726-0000.mp3", "121-121726-0001.mp3", "121-121726-0002.mp3", 
+    "121-121726-0003.mp3", "121-121726-0004.mp3", "121-121726-0005.mp3", 
+    "121-121726-0006.mp3", "121-121726-0007.mp3", "121-121726-0010.mp3", 
+    "121-121726-0011.mp3", "121-121726-0012.mp3", "121-121726-0013.mp3", 
+    "121-121726-0014.mp3", "121-123852-0000.mp3", "121-123852-0001.mp3", 
+    "121-123852-0002.mp3", "121-123852-0003.mp3", "121-123852-0004.mp3", 
+    "121-123859-0000.mp3", "121-123859-0001.mp3", "121-123859-0002.mp3", 
+    "121-123859-0003.mp3", "121-123859-0004.mp3", "4507-16021-0000.mp3", 
+    "4507-16021-0001.mp3", "4507-16021-0002.mp3", "4507-16021-0003.mp3", 
+    "61-70968-0000.mp3", "61-70968-0001.mp3", "61-70968-0002.mp3", 
+    "61-70968-0003.mp3", "61-70968-0004.mp3", "61-70968-0005.mp3", 
+    "61-70968-0006.mp3", "61-70968-0007.mp3", "61-70968-0008.mp3", 
+    "61-70968-0009.mp3", "61-70968-0010.mp3", "61-70968-0011.mp3", 
+    "61-70968-0012.mp3", "61-70968-0013.mp3", "61-70968-0014.mp3", 
+    "61-70968-0015.mp3", "61-70968-0016.mp3", "61-70968-0017.mp3", 
+    "61-70968-0018.mp3", "61-70968-0019.mp3", "61-70968-0020.mp3", 
+    "61-70968-0021.mp3", "61-70968-0022.mp3", "61-70968-0023.mp3", 
+    "61-70968-0024.mp3", "61-70968-0025.mp3", "61-70968-0026.mp3", 
+    "61-70968-0027.mp3", "61-70968-0028.mp3", "61-70968-0029.mp3", 
+    "61-70968-0030.mp3", "61-70968-0031.mp3", "61-70968-0032.mp3", 
+    "61-70968-0033.mp3", "61-70968-0034.mp3", "61-70968-0035.mp3", 
+    "61-70968-0036.mp3", "61-70968-0037.mp3", "61-70968-0038.mp3", 
+    "61-70968-0039.mp3", "61-70968-0040.mp3", "61-70968-0041.mp3", 
+    "61-70968-0042.mp3", "61-70968-0043.mp3", "61-70968-0044.mp3", 
+    "61-70968-0045.mp3", "61-70968-0046.mp3", "61-70968-0047.mp3", 
+    "61-70968-0048.mp3", "61-70968-0049.mp3", "61-70968-0050.mp3", 
+    "61-70968-0051.mp3", "61-70968-0052.mp3", "61-70968-0053.mp3", 
+    "61-70968-0054.mp3", "61-70968-0055.mp3", "61-70968-0056.mp3", 
+    "61-70968-0057.mp3", "61-70968-0061.mp3", "61-70968-0062.mp3", 
+    "61-70970-0000.mp3", "61-70970-0001.mp3", "61-70970-0002.mp3", 
+    "61-70970-0003.mp3", "61-70970-0004.mp3", "61-70970-0005.mp3", 
+    "61-70970-0006.mp3", "61-70970-0007.mp3", "61-70970-0008.mp3", 
+    "61-70970-0009.mp3", "61-70970-0010.mp3", "61-70970-0011.mp3", 
+    "61-70970-0012.mp3", "61-70970-0013.mp3"
+]
+args = None
```

Comment:
please remove this from global scope.  The init method from the test method can receive the parsed arguments

Reply:
Are you talking about "args", or "TEST_FILES", or both?

Reply:
Never mind, it's resolved.